astro/src/content/docs/extend/examples/modeling-organizations.mdx

```diff
@@ -0,0 +1,264 @@
+          "jwtConfiguration": {
+            "enabled": false,
+            "timeToLiveInSeconds": 60
+          },
+          "lastUpdateInstant": 1518962408732,
+          "name": "Customers"
+        }
+      },
+      "id": "8174f72f-5ecd-4eae-8de8-6fef597b3473",
+      "insertInstant": 1595361142929,
+      "lastUpdateInstant": 1595361143121,
+      "permissions": [
+        "Admin"
+      ],
+      "userId": "7174f72f-5ecd-4eae-8de8-7fef597b3473"
+    }
+  ],
+  "total": 1
+}
+```
+
+You can see that the permissions are available in the response. You can extract the permissions and use them to authorize actions that the user takes or APIs they are calling. In this example, the User has the `Admin` permissions, which means they are likely allowed to do anything they wish.
+
+The `grant` object, or some portion of it, can be stored in any number of locations for easy access. You could store it in a cookie (encrypted is preferred), a server-side session, or a database. The [Search Grants](/docs/v1/tech/apis/entity-management/grants#search-for-grants) is designed to be queried repeatedly and returned quickly, so depending on the scale of the application and where FusionAuth is deployed, you could also query this API for each request. This has the benefit of ensuring that the User's permissions are always the most current version.
+
+## Multiple Organizations
+
+The final component that will be covered in this guide is handling multiple organizations for a single user. AwesomeCRM works with many partners and often these partners work with many clients. This means that a user at a partner might be working with multiple organizations at the same time. To handle this scenario, you need to ensure you can handle multiple results from the [Retrieve Grants](/docs/v1/tech/apis/entity-management/grants#retrieve-grants) API.
+
+Here's an example of the response from the [Retrieve Grants](/docs/v1/tech/apis/entity-management/grants#retrieve-grants) API when a user has multiple organizations (many properties have been trimmed for brevity):
+
+```json
+{
+  "grants": [
+    {
+      "entity": {
+        "name": "Pied Piper",
+        ...
+      },
+      "permissions": [
+        "Admin"
+      ],
+      ...
+    },
+    {
+      "entity": {
+        "name": "Hooli",
+        ...
+      },
+      "permissions": [
+        "Sales"
+      ],
+      ...
+    },
+    {
+      "entity": {
+        "name": "Aviato",
+        ...
+      },
+      "permissions": [
+        "Billing",
+        "Reports"
+      ],
+      ...
+    }
+  ],
+  "total": 3
+}
+```
+
+This response indicates that the User belongs to 3 organizations and has various permissions for each. To manage this in your AwesomeCRM application, you can present an option to the user when they log in, to select the company they'd like to work on initially. This option screen might look like this:
+
+![Company Selection](/img/guides/organizations/accountselectp.png)
```

Comment:
This image is very different than the Create Entity image.  The URL should read local.fusionauth.io and the user should probably be Juan Yang again.  Most of the docs have the dark menu bar so I would update that or update the other image as well so it at least matches on the page.  I'd also update the company names to align with the results of the json. - "Pied Piper", "Hooli", and "Aviato".

Reply:
It just hit me that this is not the admin ui.  Please ignore the comments about the url and dark menu bar.  Still think it should be Jian Yang and have the company names line up if possible.